tool/tctl/common/resource_command.go

```diff
@@ -2426,6 +2427,16 @@ func (rc *ResourceCommand) getCollection(ctx context.Context, client auth.Client
 			return nil, trace.Wrap(err)
 		}
 		return &serverInfoCollection{serverInfos: serverInfos}, nil
+	case types.KindAccessList:
```

Comment:
This blurb isn't totally needed because `tctl acl ls/get` exists. I'm not opposed to this though.

Reply:
`title create` / `title get` work in tandem; if one is implemented, I think the second should also be. Subcommands can provide more straightforward ways of interacting with the resource, but get/create/edit/delete are the baseline for everything.

This was not standardized when accesslists were implemented but has been formalized since in RFD-0153: https://github.com/gravitational/teleport/blob/master/rfd/0153-resource-guidelines.md#new-resource-checklist